src/transport_performance/gtfs/validation.py

```diff
@@ -116,6 +131,20 @@ def __init__(
             raise ValueError(f"`units` accepts metric only. Found: {units}")
 
         self.feed = gk.read_feed(gtfs_pth, dist_units=units)
+        self.gtfs_path = gtfs_pth
+
+    def get_gtfs_files(self) -> list:
```

Comment:
I can think of 2 other conditions we would want to check for raise:

1. Fewer files than expected
2. Additional files than expected (perhaps a warning is more appropriate here if we can test it without having it show up in test reports)

As logic not implemented, this can be a feature request. Would you agree?

Reply:
Yes I agree with this. From my understanding you're referencing the actual gtfs files e.g., calendar.txt. I'm happy if we want to open this as a feature request, it would be a nice layer of protection to have.  
`gtfs-kit` may already have this functionality which we would need to confirm or deny first.

Reply:
Agreed to raise as issue, can I leave this with you?